controllers/front/SmailyCustomerCron.php

```diff
@@ -126,6 +126,7 @@ private function removeUnsubscribers($limit = 1000)
 
             // Stop if error.
             if (!isset($unsubscribers['success'])) {
+                $this->module->logMessageWithSeverity("Failed fetching unsubscribers.", 3);
```

Comment:
Just use 

```php
$this->module->logErrorWithFormatting("[your message here]");
```
here.

It will work because the call will be translated to:

```php
sprintf("[your message here");

// This is what these 2 lines do:
// $args = func_get_args();
// $message = call_user_func_array('sprintf', $args);
```

and `sprintf` needs at least one argument passed as long as it does not contain any formatting fields like `%s` or `%d`.